### ISSUE TITLE
OAI-PMH Harvesting

### ISSUE THREAD

**Issue by jacimize** [Esri/geoportal-server#258]
I am trying to register a resource on my geoportal instance using the OAI protocol type. I have not done this before and I was wondering if there were any pointers? I am not seeming to have success. 
Host URL: http://repository.library.noaa.gov/fedora/oai?verb=ListRecords&metadataPrefix=oai_dc
Prefix: oai_dc

**Comment by jacimize** [comment]
any updates on this issue or how we may resolve it? We made sure to add the OAI schema but still no luck.

**Comment by mhogeweg** [comment]
hi, in this case we suspect there is a bug related to the handling of the [robots.txt](https://repository.library.noaa.gov/robots.txt) file. We implemented this to respect the source's wishes to limit what crawlers access. The respecting of robots.txt is a goodwill gesture from the crawler. 

In this case we suspect something is going wrong when parsing the file. You can however harvest the site (we tested this internally) by turning off the setting to respect robots.txt. You can change this in the harvest registration page.

**Comment by jacimize** [comment]
Thanks, will test that ASAP.

**Comment by simongis** [comment]
Can someone confirm if Geoportal Server supports OAI-PMH harvesting?

**Comment by mhogeweg** [comment]
yes, geoportal supports OAI-PMH harvesting. this is the case for both Geoportal Server [1.x](https://github.com/Esri/geoportal-server/wiki/How-to-Publish-Resources#Register_a_resource_on_the_network) and [2.x](https://github.com/Esri/geoportal-server-harvester/wiki/Features)

**Comment by simongis** [comment]
Thanks @mhogeweg
